src/ui/server/controllers/insurance/policy/loss-payee-financial-details-international/validation/index.test.ts

```diff
@@ -0,0 +1,22 @@
+import validation from '.';
+import validationRules from './rules';
+import FIELD_IDS from '../../../../../constants/field-ids/insurance/policy';
+import combineValidationRules from '../../../../../helpers/combine-validation-rules';
+
+const {
+  LOSS_PAYEE_FINANCIAL_UK: { ACCOUNT_NUMBER },
```

Comment:
need `LOSS_PAYEE_FINANCIAL_INTERNATIONAL` field IDs here

Reply:
Actually don't need and field IDs in such validation tests with an empty body - removed